objects/obj_player/Alarm_1.gml

```diff
@@ -0,0 +1,13 @@
+if (buff_active) {
+    // Buff的持续时间已经结束，取消变化
```

Comment:
for consistency and understanding throughout the team, we may want to keep the comments in English.

Reply:
sorry about it, I forgot to translate into English
Fixed